bot/cogs/base_converter_cog.py

```diff
@@ -0,0 +1,129 @@
+import discord
+import discord.ext.commands as commands
+
+from bot.consts import Colors
+
+class BaseConverterCog(commands.Cog):
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.group()
+    async def baseconvert(self, ctx):
+        """
+        A simple base converter that takes in a base number an a value, then displays the value in binary, octal, decimal, and hexadecimal
+        Examples:
+            baseconvert(bin 0b11010101)
+            baseconvert(oct 0o1234)
+            baseconvert(dec 4567)
+            baseconvert(hex 0xAF13)
+        """
+        if ctx.invoked_subcommand is None:
+            await ctx.send('Invalid Subcommand!')
+
+    @baseconvert.command()
+    async def bin(self, ctx, number : str):
+        number = int(number, 2)
+        d = int(number)
+        o = oct(number)	
+        h = hex(number)
+        if number in range(1114112): 
+            a = chr(number)
+        else:
+            a = 'Unicode representation not found.'
+
+        embed = discord.Embed(title='Conversions', description =f'Numerical Conversions of {str(number)}', color = Colors.ClemsonOrange)
```

Comment:
you dont need str() in an f string its redundant, its implicitly called on the expression in the brackets {}